api/class-wc-calypso-bridge-send-invoice-controller.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * REST API WC Calypso Bridge Send Invoice
+ *
+ * Adds an endpoint to trigger sending an order invoice email
+ *
+ * @author   Automattic
+ * @category API
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * @package WooCommerce/API
+ */
+class WC_Calypso_Bridge_Send_Invoice_Controller extends WC_REST_Controller {
+
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc/v3';
+
+	/**
+	 * Route base.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'orders/send_invoice';
+
+	/**
+	 * Post type.
+	 *
+	 * @var string
+	 */
+	protected $post_type = 'shop_order';
+
+	/**
+	 * Register Currency route
+	 */
+	public function register_routes() {
+		register_rest_route( $this->namespace, '/' . $this->rest_base . '/(?P<id>[\d]+)' , array(
```

Comment:
I think this endpoint should follow the other order resources, and be `/orders/<id>/send_invoice` (see [notes](https://github.com/woocommerce/woocommerce/blob/master/includes/api/v1/class-wc-rest-order-notes-controller.php#L37), for example)